### ISSUE TITLE
some questions for baseline model

### ISSUE THREAD

**Issue by RQsky** [jayleicn/VideoLanguageFuturePred#10]
Good brother, hello. In our last conversation, I hope you can tolerate some inappropriate words.
I have made a lot of progress on the road to achieve the baseline. I have achieved all the points that I think are important, but the score just can't be improved, and I have never broken through 0.55.
The model I made is somewhat different from your baseline. For example, I use slowfast feature instead of resnetx-101, and the embedding part may be different, and I did not add CLS mark before the video sequence, so I use the maximum pooling result prediction output by the fusion layer instead of CLS.
There is also the OOM problem. My solution is to input a shorter sequence and use cumulative loss to increase gradient_accumulation_steps to simulate a larger batch size.
But I think these are not very important points, but the score is so bad that I am very confused.
If you have time, please help me look at my code
My model address is: [https://github.com/RQsky/vlep/blob/master/train_vlep.py#L131](https://github.com/RQsky/vlep/blob/master/train_vlep.py#L131)
My model config is: [https://github.com/RQsky/vlep/blob/master/config/train-vlep-baseline.json](https://github.com/RQsky/vlep/blob/master/config/train-vlep-baseline.json)
I use the same environment as the docker provided by VALUE: [https://github.com/VALUE-Leaderboard/StarterCode](https://github.com/VALUE-Leaderboard/StarterCode)
Use roberta weight initialization of hugging face: [https://huggingface.co/roberta-base/blob/main/pytorch_model.bin](https://huggingface.co/roberta-base/blob/main/pytorch_model.bin)
Of course, I know you must be very busy. In fact, I would be very grateful for simply answering some of my questions
